tests/vm_test.go

```diff
@@ -971,309 +969,149 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
+		vm.APIVersion = version
+		vm, err := virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
 
-			By("Creating VM with DataVolumeTemplate entry with k8s client binary")
-			_, stdErr, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "delete", "vm", vmi.Name)
-			Expect(err).To(HaveOccurred())
-			Expect(strings.HasPrefix(stdErr, "Error from server (NotFound): virtualmachines.kubevirt.io")).To(BeTrue(), "should fail when deleting non existent VM")
-		})
+		By("Waiting for VMI to start")
+		Eventually(ThisVMIWith(vm.Namespace, vm.Name), 120*time.Second, 1*time.Second).Should(BeRunning())
 
-		Context("as ordinary OCP user trough test service account", func() {
-			var testUser string
+		By("Ensuring pod is running")
+		expectedPodName := getExpectedPodName(vm)
+		Expect(ThisPodWith(vm.Namespace, expectedPodName)).To(BeRunning())
 
-			BeforeEach(func() {
-				testUser = "testuser-" + uuid.NewString()
-			})
+		By("Checking that VM is running")
+		Expect(ThisVM(vm)).To(BeRunning())
+	},
+		Entry("with v1 api", "kubevirt.io/v1"),
+		Entry("with v1alpha3 api", "kubevirt.io/v1alpha3"),
+	)
 
-			Context("should succeed with right rights", func() {
-				BeforeEach(func() {
-					// kubectl doesn't have "adm" subcommand -- only oc does
-					clientcmd.SkipIfNoCmd("oc")
-					By("Ensuring the cluster has new test serviceaccount")
-					stdOut, stdErr, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "create", "user", testUser)
-					Expect(err).ToNot(HaveOccurred(), "ERR: %s", stdOut+stdErr)
-
-					By("Ensuring user has the admin rights for the test namespace project")
-					// This simulates the ordinary user as an admin in this project
-					stdOut, stdErr, err = clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "adm", "policy", "add-role-to-user", "admin", testUser)
-					Expect(err).ToNot(HaveOccurred(), "ERR: %s", stdOut+stdErr)
-				})
-
-				AfterEach(func() {
-					stdOut, stdErr, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "adm", "policy", "remove-role-from-user", "admin", testUser)
-					Expect(err).ToNot(HaveOccurred(), "ERR: %s", stdOut+stdErr)
-
-					stdOut, stdErr, err = clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "delete", "user", testUser)
-					Expect(err).ToNot(HaveOccurred(), "ERR: %s", stdOut+stdErr)
-				})
-
-				It("[test_id:2839]should create VM via command line", func() {
-					By("Checking VM creation permission using k8s client binary")
-					stdOut, _, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "auth", "can-i", "create", "vms", "--as", testUser)
-					Expect(err).ToNot(HaveOccurred())
-					Expect(strings.TrimSpace(stdOut)).To(Equal("yes"))
-				})
-			})
+	It("[test_id:264]should create and delete a VM successfully", decorators.Conformance, func() {
```

Comment:
Not sure about the value of this test. Isn't that like the most basic functionality used in many other tests already?